### ISSUE TITLE
Major Issue, Infinate "was not found in Stats database!" for all UUID.

### ISSUE THREAD

**Issue by TomLewis** [Staartvin/PluginLibrary#2]
Why can I never, ever update Autorank without there being a major issue.

Today for the first time in must be a year, I decided to update Autorank 4.0.7 to 4.1, which in turn added this "PluginLibrary", I use Stats3 still as Statz has never been able to convert without issue, and it tried to take over 48h for a simple SQL query.

Now the issue is this, for some unknown reason it started spammign the console with these, and there infinite, like I have over 100,000 users, My Stats3 and Autorank 4.0.7 has worked fine up until today when I updated AR and added plugin Library, why is it doing this. its creating significant lag.
```
[00:19:09] [Craft Scheduler Thread - 1140/INFO]: [PluginLibrary] UUID 'aaadd056-a743-438e-a4ac-b20d521cc6e3' was not found in Stats database!
[00:19:09] [Craft Scheduler Thread - 1140/INFO]: [PluginLibrary] UUID 'ed1a13d0-941c-4745-957f-7dca30209616' was not found in Stats database!
[00:19:09] [Craft Scheduler Thread - 1140/INFO]: [PluginLibrary] UUID 'c6bee325-81cb-4290-a41e-0a2ded9a5776' was not found in Stats database!
[00:19:09] [Craft Scheduler Thread - 1156/INFO]: [PluginLibrary] UUID '0f80ca48-890b-455f-8701-045a37e9fc44' was not found in Stats database!
[00:19:11] [Craft Scheduler Thread - 1156/INFO]: [PluginLibrary] UUID '8768c383-e50f-46a6-afe8-ab1d5b8a54f3' was not found in Stats database!
[00:19:11] [Craft Scheduler Thread - 1140/INFO]: [PluginLibrary] UUID '73fdf26d-e742-475d-b90b-1448b0f6e1ef' was not found in Stats database!
[00:19:11] [Craft Scheduler Thread - 1156/INFO]: [PluginLibrary] UUID 'bf4dbc38-0fe2-4e4a-80df-ee7caf2bc227' was not found in Stats database!
[00:19:11] [Craft Scheduler Thread - 1140/INFO]: [PluginLibrary] UUID 'a02950d6-edeb-4e9c-9917-f6e6f83c42d7' was not found in Stats database!
```

**Comment by Staartvin** [comment]
Hmm, I remember having tested your large database which ought to import your data from   Stats to Statz just fine.

**Comment by Staartvin** [comment]
Then, after importing has worked, we have to check if you can remove Stats3. Lastly, we turn on Autorank and it should to the job.

**Comment by TomLewis** [comment]
That's not a fix, it would be easier for me to just backdate so its working. I cant trust the conversion process to;
1. Work without loosing data
2. Convert in a reasonable time, an SQL query shouldn't ever take as long as Statz is taking, Its like 2 days or something insane, it shouldn't take longer than 10 mins MAX!

That's a whole other issue though for another time, right now it needs to just be working without crashing my server. Im using Stats3, I dont have the time to play about with the conversion and make sure the data isn't all broken, lost or whatever, its a massive time consuming job. 

The whole point of your "PluginLibrary" is to support existing stats, thats what im doing! Im using stats and im reporting a bug! why is it saying this? can you please just look into the bug? else I think ill just backdate.

**Comment by Staartvin** [comment]
I would love to obtain your mysql database again, try to rerun the process and see how long it takes. I've spent last year working on this process and I believe it was fixed, so I can test it in my time and report back.

**Comment by TomLewis** [comment]
Yeah, its not reading any data from stats, when I do /track 3 it displays -1, so its not getting any data.
![2018-04-13_13 03 04](https://user-images.githubusercontent.com/144046/38733860-765c40c6-3f1b-11e8-8e0c-3cc85cbe18b3.png)

I did /version stats and my version is 3.1.1-RC, where as your pluginLibrary says v3.0.2-382, could that be the issue? That the methods used to call Stats are different? 

If not, I will just backdate and not upgrade.

**Comment by Staartvin** [comment]
Would you mind sending me a copy of your stats database (via discord) for example?
